plugin/src/main/java/org/wildfly/swarm/plugin/maven/RepositorySystemSessionWrapper.java

```diff
@@ -175,4 +190,24 @@ public RepositoryCache getCache() {
     private RepositorySystemSession delegate;
 
     private DependencyGraphTransformer transformer;
+
+    private boolean excludeSwarm;
+
+    public class SwarmExcludedTraverser implements DependencyTraverser
+    {
+
+       @Override
+       public boolean traverseDependency(Dependency dependency)
+       {
+          return !SWARM_GROUP_ID.equals(dependency.getArtifact().getGroupId());
```

Comment:
In the other places, we've done the possibly-sub-optimal route of figuring out if a jar is a swarm fraction jar by the presence of META-INF/fraction-manfiest.yml at this point.